tests/autoyast/autoyast_verify.pm

```diff
@@ -92,7 +89,7 @@ sub test_flags {
     # 'fatal' - whole test suite is in danger if this fails
     # 'milestone' - after this test succeeds, update 'lastgood'
     # 'important' - if this fails, set the overall state to 'fail'
-    return {fatal => 1};
+    return {important => 1};
```

Comment:
I am the new guy, you are the expert. I ask questions. If you want to be kind, you give answers. If you are smart you ask questions so that I find the solution myself. Good job, I got it, you are smart, yes log uploading moved ;-)
I doesn't help me to understand why it moved, though (or why @nadvornik moved it).